<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreditWindow</class>
 <widget class="QMainWindow" name="CreditWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Кредитный калькулятор</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonClean">
    <property name="geometry">
     <rect>
      <x>334</x>
      <y>301</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Отчистить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCalc">
    <property name="geometry">
     <rect>
      <x>429</x>
      <y>301</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Рассчитать</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>198</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Общая выплата</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>233</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Переплата по кредиту</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>268</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Ежемесячный платеж</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelOverPayment">
    <property name="geometry">
     <rect>
      <x>243</x>
      <y>236</y>
      <width>201</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelTotalPayments">
    <property name="geometry">
     <rect>
      <x>242</x>
      <y>198</y>
      <width>201</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelMonthlyPayment">
    <property name="geometry">
     <rect>
      <x>243</x>
      <y>268</y>
      <width>201</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>128</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Тип ежемесячных платежей</string>
    </property>
   </widget>
   <widget class="QComboBox" name="paymentsType">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>128</y>
      <width>211</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Аннуитетный</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Дифференцированный</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="percentageRate">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>88</y>
      <width>211</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>20</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>88</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>88</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Процентная ставка</string>
    </property>
   </widget>
   <widget class="QComboBox" name="typePeriod">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>48</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>месяцев</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>лет</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="creditPeriod">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>48</y>
      <width>211</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>24</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>49</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Срок кредита</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="creditAmount">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>8</y>
      <width>211</width>
      <height>25</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly|Qt::ImhNoTextHandles</set>
    </property>
    <property name="text">
     <string>100000</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>8</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>₽</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>8</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Сумма кредита</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
